tests/setup.js

```diff
@@ -67,7 +67,11 @@ beforeEach(() => {
 
 afterEach(() => {
     if (logs.length > 0 || warns.length > 0 || errors.length > 0) {
-        throw new Error('Unexpected console logs' + logs + warns + errors);
+        const message = 'Unexpected console logs' + logs + warns + errors;
+        if (message.includes('componentWillReceiveProps')) {
```

Comment:
What component is printing this error? Was it because of a `componentWillReceiveProps` or an `UNSAFE_componentWillReceiveProps`, since the latter is what we should be using now

Reply:
Loads and loads of them. Hence me just hiding the error rather than trying to fix it.

Reply:
I guess if it's printing errors here, that means they'll be printed in dev mode too so that we'll be more motivated to actually fix them 😛